troubleshooting/debugging.md

```diff
@@ -1010,6 +1023,152 @@ In cases of high log output rate from pods, or a large number of pods per node,
 
 Consequently, we recommend carefully monitoring your [EBS metrics in CloudWatch](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-io-characteristics.html) and considering switching to [storage.type memory](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/oomkill-prevention#case-1-memory-buffering-only-default-or-storagetype-memory).
 
+#### EKS Tail Multiline Configuration
+
+Fluent Bit supports multiline log concatenation for split container logs. Most container runtimes split logs after some limit, usually 16KB. This means a single newline delimitted statement emitted by your application may be split into multiple log lines in the log file. Fluent Bit can recombine the messages. Fluent Bit supports K8s on Docker and K8s on CRI. 
+
+For example, imagine the max log size in the container runtime was 10 bytes, then if the pod emits the log line "hello world!", in the log file there would be two lines:
+```
+2023-05-19T20:13:35.24481424Z stdout P Hello worl
+2023-05-19T20:13:35.24481424Z stdout F d!
+```
+
+The recommended Fluent Bit tail configuration below will read this as a single log event:
+
+```
+{
+    time": "2023-05-19T20:13:35.24481424Z",
+    "stream": "stderr",
+    "_p": "P",
+    "log": "Hello world!",
+}
+```
+
+See the [Fluent Bit documentation](https://docs.fluentbit.io/manual/pipeline/inputs/tail#multiline-and-containers-v1.8). See the Kubernetes repo for the [spec of the log file format](https://github.com/kubernetes/design-proposals-archive/blob/main/node/kubelet-cri-logging.md).
+
+```
+[INPUT]
+    name              tail
+    path              /var/log/containers/*.log
+    multiline.parser  docker, cri
+```
+
+#### Adding custom multiline parsing rules on top of CRI multiline parsing: only one parser is applied
```

Comment:
So if a user is on Docker instead of CRI, then would only one multiline parser be allowed, the one processed by Fluent Bit? 
If so, can we clarify this up in the `Limitation: No more than 2 multiline parsers per log tag` section?

Reply:
Its the same for docker and cri... AFAIK most k8s users are on CRI now... will change to "CRI or Docker"